src/generators/concretes/proposedBlock.generator.ts

```diff
@@ -0,0 +1,36 @@
+import { ObjectHasher } from "@blockr/blockr-crypto";
+import { Block, Transaction } from "@blockr/blockr-models";
+import { inject, injectable } from "inversify";
+import { ConstantStore } from "../../stores";
+import { BlockGenerator } from "../abstractions/block.generator";
```

Comment:
Simplify this import & alphabetically order imports

```suggestion
import { BlockGenerator } from "../abstractions/block.generator";
```